ert_gui/tools/event_viewer/panel.py

```diff
@@ -0,0 +1,50 @@
+from qtpy.QtWidgets import QVBoxLayout, QPlainTextEdit
+
+from qtpy import QtCore
+import logging
+
+
+class GUILogHandler(logging.Handler, QtCore.QObject):
+    """
+    Log handler which will emit a qt signal every time a
+    log is emitted,
```

Comment:
comma? Something more intended to be written?

Reply:
Not as far as I remember :grimacing: I have removed it